start.sh

```diff
@@ -4,14 +4,12 @@
 cd von-network/
 VONIMAGE=`docker images | grep von-network`
 [[ ! -z "$VONIMAGE" ]] || ./manage build
-./manage start
+./manage start --wait
 cd ../
 
-# WAIT
-echo -en "\n\nWaitingForVonWebserver"; RES=""; while [[ -z "$RES" ]]; do sleep .1; RES=$(curl -sf localhost:9000/genesis 2>&1); echo -n .; done
-
 # START F2P ARIES CLOUD AGENT
-docker-compose -f ./docker/docker-compose.yaml -p veritable-demo up --build -d
+docker compose -f ./docker/docker-compose.yaml pull 
```

Comment:
Just FIY, `./manage` still uses `docker-compose` instead of `docker compose`. I don't think this makes any difference but it makes sense to keep this in mind for the future .. who knows , this info might end up being useful